server/src/main/java/com/flipkart/varadhi/db/ZKPathUtils.java

```diff
@@ -10,6 +10,10 @@ public static String getVaradhiTopicPath(String topicName) {
         return constructPath(VARADHI_TOPIC_NAME, topicName);
     }
 
+    public static String getVaradhiProjectPath(String projectName) {
+        return constructPath(TOPIC_RESOURCE_NAME, projectName);
+    }
+
     public static String getTopicResourceFQDN(String projectName, String topicName) {
         return String.join("/", projectName, topicName);
```

Comment:
There is a bug in the code I checked in. This should not be path separator in above. But a name separator.  We can define a constant and use it accordingly. 

Since DB package is missing UTs, can we add some tests to start with as well.